gobblin-salesforce/src/main/java/org/apache/gobblin/salesforce/SalesforceSource.java

```diff
@@ -504,6 +534,14 @@ void add(Histogram histogram) {
       totalRecordCount += histogram.totalRecordCount;
     }
 
+    HistogramGroup get(int idx) {
```

Comment:
Why is this required? I don't see anywhere where the null result is checked.

Reply:
I use this to get the latest group.